src/btree/bt_curnext.c

```diff
@@ -348,7 +348,7 @@ __cursor_row_next(
     key = &cbt->iface.key;
     page = cbt->ref->page;
     session = CUR2S(cbt);
-    prefix_search = prefix != NULL && F_ISSET(&cbt->iface, WT_CURSTD_PREFIX_SEARCH);
```

Comment:
Nitpick: thinking to rename the `prefix_search` to `prefix_key_search`

Reply:
I am fine with either, let's see what's Luke's preferences.

Reply:
Please, review again as it was decided to keep `prefix_search` instead of `prefix_key` and the changes are not the same.